woocommerce/google-listings-and-ads
# Fork `ChooseAudience` component and use it in `EditFreeCampaign`

### Changes proposed in this Pull Request:

- Duplicate ChooseAudience component,
- remove saving strategy from it
- move reusable parts to `.~/components/free-listings`
- make step header configurable
- use it in EditFreeCampaign


### Screenshots:

![image](https://user-images.githubusercontent.com/17435/110356551-ac2cae80-803a-11eb-8892-fb895189d02f.png)

https://user-images.githubusercontent.com/17435/110356857-f4e46780-803a-11eb-9801-ee57a6f36968.mp4




### Detailed test instructions:

1. Visit https://gla1.test/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fedit-free-campaign&programId=123
2. Make sure step Content "Configure your product listings" step title looks as in [Figma](https://www.figma.com/file/jZUpa8eTrnrK1Lwt2ry7zk/Native-Google-Integration?node-id=4162%3A0)

### Still missing
Things to be addressed in future PRs:

- fix topbar (waits for [the PR that makes it customizable](https://github.com/woocommerce/google-listings-and-ads/pull/301) to be merged)
- save changes on the last step
- prompt on leave
